pages/alerts.rst

```diff
@@ -240,6 +240,50 @@ Here is an example of the payload included in a notification::
 .. image:: /images/alerts_http_notification.png
 
 
+.. _alert_notification_pagerduty:
+
+PagerDuty alert notification
+----------------------------
+
+The PagerDuty alert notification allows you create new incidents in PagerDuty in response
+to Events in your Graylog server.
+
+.. note:: PagerDuty alert notifications are intended as a replacement for the PagerDuty
+   integration previously available from 
+   `Graylog Labs <https://github.com/graylog-labs/graylog-plugin-pagerduty>`__.  If you 
+   are using the Graylog Labs PagerDuty integration, you should migrate to the officially
+   supported PagerDuty alert notifications at your earliest convenience.
+
+.. important:: In order to use PagerDuty alert notifications, you must create a new 
+   integration in PagerDuty to obtain an integration routing key:     
+   `PagerDuty Documentation <https://developer.pagerduty.com/docs/events-api-v2/overview/#getting-started>`__ 
+
```

Comment:
Do you need a new integration if you are already using the PagerDuty Integration? We should probably state that if so.

Reply:
Yeah, I can clarify that you only need to create a new integration if you don't already have one.

Reply:
Updated text

![Screen Shot 2020-10-29 at 11 08 22 AM](https://user-images.githubusercontent.com/6466251/97592486-166f7b80-19d7-11eb-8d45-b416f26e0683.png)